### PR TITLE
chore: setup SQL review CI for Bytebase

### PR DESCRIPTION
**Pull request by Francois-Werbrouck**
This pull request is auto-generated by Bytebase for GitOps workflow.

### CODE REVIEW

File: .github/workflows/bytebase-sql-review.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+on: [pull_request]
+jobs:
+  bytebase-sql-review:
+    runs-on: ubuntu-latest
+    name: SQL Review
+    steps:
+      - name: SQL advise
+        run: |
+          API="https://bytebase.inspection.alpha.canada.ca/hook/sql-review/c033cbf2-f0e6-4c1a-89ba-ced209b06fba-1712080200"
```
Comment (rngadam) [inline .github/workflows/bytebase-sql-review.yml:9]: this could/should also be a secret?

Comment (rngadam) [reply .github/workflows/bytebase-sql-review.yml:9]: also naming BYTEBASE_API and BYTEBASE_TOKEN would make it more obvious what these constants are.

**Review by rngadam — CHANGES_REQUESTED** [review]

**Comment by SonOfLope** [comment]
Updated to call a reusable workflow once this PR is merged : https://github.com/ai-cfia/github-workflows/pull/114

File: .github/workflows/bytebase-sql-review.yml (hunk 1 above)
Comment (SonOfLope) [reply .github/workflows/bytebase-sql-review.yml]: done in reusable workflow

**Comment by SonOfLope** [comment]
fyi theres no checks because current state of PR makes the workflow invalid since its pointing to the reussable workflow that hasnt been merged yet (so it resolves into an error of inexistant workflow)

**Review by rngadam — CHANGES_REQUESTED** [review]
does anything prevent us from pointing to the branch in .github/workflows?

**Comment by SonOfLope** [comment]
> does anything prevent us from pointing to the branch in .github/workflows?

No, its just a preference of being patient until the workflow is merged. The workflow itself has been tested before doing the changes with a reusable workflow.

**Review by rngadam — APPROVED** [review]

**Pull request merged**